### PR TITLE
[[ Bug 17895 ]] Make dgText[true] less ambiguous

### PR DESCRIPTION
**Pull request by jameshale**

### CODE REVIEW

File: Documentation/dictionary/datagrid.lcdoc [hunk 1]
```diff
@@ -1650,31 +1650,41 @@ making life easier for some folks there is a dgText property. The dgText
...
 pFirstLineContainsHeaders. Normally you should set this property to true
 and provide the header that maps each item of each line to a specific
-column. Note that if pFirstLineContainsHeaders is true then the columns
-must already exist in your data grid table in order to be displayed. If
-pFirstLineContainsHeaders is false then the columns property of the data
+column. 
+
+>Note that if pFirstLineContainsHeaders is true then the columns
+>must already exist in your data grid table in order to be displayed. 
+>Setting pFirstLineContainsHeaders to true does not create the headers for you.
```
Comment (livecodeali) [inline Documentation/dictionary/datagrid.lcdoc:1671]: Can you format this as 

```
>*Note:* ...
```

please?

**Comment by livecodeali** [comment]
Thanks for this! The only other thing that may cause an issue here is the filename of the bugfix note - it should have a `.md` file extension.

**Comment by livecodeali** [comment]
@livecode-vulcan review ok 6b8a8ac

**Comment by livecode-vulcan** [comment]
💙 review by @livecodeali ok 6b8a8ac917f2670119d0abd145c1b060512751a2

**Pull request merged**